File: easy/matrix/Row With Maximum Ones.py
class Solution(object):
    def rowAndMaximumOnes(self, mat):
        row,c=0,0
        for i in range(len(mat)):
            one=0
            for j in range(len(mat[0])):
                if mat[i][j]==1:
                    one+=1
            if one>c:
                c=one
                row=i

        return [row,c]

File: easy/matrix/Row With Maximum Ones.java
class Solution {
    public int[] rowAndMaximumOnes(int[][] mat) {
        int row = 0, c = 0;

        for (int i = 0; i < mat.length; i++) {
            int ones = 0;
            for (int j = 0; j < mat[i].length; j++) {
                if (mat[i][j] == 1) {
                    ones++;
                }
            }
            if (ones > c) {
                c = ones;
                row = i;
            }
        }
        return new int[]{row, c};
    }
}
